docs/extension/guides/create-backup.md

```diff
@@ -51,10 +52,4 @@ The 2FA recovery system for Universal Profiles on [my.universalprofile.cloud](ht
 
 ### Is there a fully decentralized backup method?
 
-If you do not trust 2FA and want an autonomous backup, users can choose the **File Export**. This process reveals the private key of the extension's controller. Please ensure that the revealed private key is stored somewhere safe.
-
-:::success Upcoming Feature
-
-Currently, the extension does not support importing profiles based on a private key. This [feature request](../../general/feature-requests.md) will soon be integrated into the Universal Profile Browser extension. A similar tool will allow you to manually add controllers from an existing private key, making it possible to add a hardware wallet as a recovery controller.
-
-:::
+If you do not trust 2FA and want an autonomous backup, you can restore your Universal Profile by [recovering from a private key](./wallet-recovery.md) add by adding additional recovery controllers using the [Wallet Recovery Setup](./set-wallet-recovery.md).
```

Comment:
what is an "autonomous backup" ?

Maybe you mean "full custody" of the backup.

or maybe "independent"